### ISSUE TITLE
readme.rst: Broken link

### ISSUE THREAD

**Issue by matmis** [esp8266/Arduino#3341]
The "Class Discription" link under "What's Inside?" in [readme.rst](https://github.com/esp8266/Arduino/blob/master/doc/esp8266wifi/readme.rst) is broken.

**Comment by devyte** [comment]
You are right. I don't know rst markup. Can you figure out the needed change and report it here? I can then fix it.

**Comment by dragondaud** [comment]
Existing: `Class Description <class-description>`__
Should be: `Class Description`_

Where a single underscore is an internal link, and the <> are not needed as the section name matches the link text. I will create a PR for this minor fix.
